src/app/zosmf-server-config/zosmf-server-config.component.ts

```diff
@@ -102,21 +119,22 @@ export class ZosmfServerConfigComponent {
   }
 
   reload(): void {
-    this.globalVeilService.showVeil();
-    this.configService
-      .getConfig()
-      .then(config => (this.config = config))
-      .then(_ => setTimeout(() => this.test(), 0))
-      .then(_ => this.globalVeilService.hideVeil())
-      .catch(err => {
-        this.globalVeilService.hideVeil()
-        let errorTitle: string = "Error";
-        let errorMessage: string = "Server configuration not found.";
-        const options = {
-          blocking: true
-        };
-          this.popupManager.reportError(ZluxErrorSeverity.ERROR, errorTitle.toString()+": "+err.status.toString(), errorMessage+"\n"+err.toString(), options);  
-        });
+      this.globalVeilService.showVeil();
```

Comment:
In general we try to avoid making whitespace-only changes. You don't *have* to fix this change, but try to avoid whitespace-only changes in the future: see if you can figure out how you ended up making whitespace-only changes here.

Reply:
Right, I see the indentation. I'll change it in a later commit, along with some other things.